Updates Vagrant Documentation

danehans opened the pull request:
Previously, the Vagrant documentation was missing a few minor
steps. This patch adds verbosity to the Vagrant documentation that
should improve developer onboarding.

eparis commented on ansible/vagrant/README.md:
```diff
@@ -1,14 +1,52 @@
+
+[Install Vagrant](https://www.vagrantup.com/downloads.html) if it's not currently installed on your system.
+
+You will need a functioning [vagrant provider](https://www.vagrantup.com/docs/providers/). Currently supported providers are openstack, libvirt, and virtualbox. Vagrant comes with VirtualBox support by default. No matter what provider you choose, you need to install the OpenStack Vagrant plugin:
```
you need to openstack plugin? that wasn't true at one point....  (and isn't supposed to be required)

danehans replied:
It was definitely required in my testing even though I was using vbox. I believe due to: https://review.openstack.org/#/c/241866/

eparis commented on ansible/vagrant/README.md:
```diff
@@ -1,14 +1,52 @@
+
+## Usage
+
+Due to [Issue 636](https://github.com/kubernetes/contrib/issues/636)], the Vagrant deployer requires the packageManager source_type. Update ~/contrib/ansible/group_vars/all.yml with the following:
```
Lets just get 636 fixed quickly   :)

danehans replied:
Agreed. I believe @rutsky and/or @stephenrlouie will be adding coreos support, so 636 will definitely need to get fixed.

eparis commented:
take out menion of 636 and I will merge. Then we get 636 fixed quickly   :)

danehans commented:
@eparis I removed the reference to 636 but kept the instructions to update the default source_type in vagrant/README.md. I also updated 636 making a note to remove this step in the instructions before closing the issue.
